### PR TITLE
Add Python package lineedit.

### PR DESCRIPTION
**Pull request by jdblischak**
I created a recipe for the PyPI package [lineedit](https://pypi.org/project/lineedit/) with grayskull version 0.8.4.

Note that this package vendors an older version of [prompt_toolkit](https://pypi.org/project/prompt-toolkit/) for backwards compatibility with Python 2. Both of the license files (MIT for this package; BSD for prompt_toolkit) are included.

xref: https://github.com/randy3k/radian/issues/244

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jdblischak** [comment]
@randy3k Please confirm below that you agree to be a maintainer of this recipe. Thanks!

**Comment by randy3k** [comment]
Thanks for working on it.

**Comment by jdblischak** [comment]
@conda-forge/staged-recipes This Python package recipe is ready for review

File: recipes/lineedit/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - lineedit
+    - lineedit.deps
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/randy3k/lineedit
+  summary: An readline library based on prompt_toolkit which supports multiple modes
```
Comment (jakirkham) [inline recipes/lineedit/meta.yaml:39]: Huh would have assumed that this means this depends on `prompt_toolkit`, but [that is not the case]( https://github.com/randy3k/lineedit/blob/4262e9852aa50b09ff3f0569e1840b003ed00584/setup.py#L33-L37 )

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks John! 😄

File: recipes/lineedit/meta.yaml (hunk 1 above)
Comment (jdblischak) [reply recipes/lineedit/meta.yaml:39]: Right, instead an older version of prompt_toolkit is directly vendored, hence the need for 2 license files below.

**Pull request merged**